pkg/storage/pebble.go

```diff
@@ -619,6 +620,9 @@ func DefaultPebbleOptions() *pebble.Options {
 		FormatMajorVersion:          MinimumSupportedFormatVersion,
 	}
 	opts.Experimental.L0CompactionConcurrency = l0SubLevelCompactionConcurrency
+	opts.Experimental.IngestSplit = func() bool {
```

Comment:
this should probably be in its own commit? perhaps not in this pr all together?

Reply:
nice catch, definitely didn't mean to make this call for storage.